### ISSUE TITLE
App spec 0.2.0

### ISSUE THREAD

**Issue by TheJaySmith** [argoproj/argo-cd#676]
Howdy,

I deployed kubeflow as a ksonnet app in Argo-CD and am getting this error. 

rpc error: code = InvalidArgument desc = Failed to list ksonnet app params: load configuration: Current app uses unsupported spec version '0.2.0' (this client only supports 0.1.0)

Is there a timeline for when 0.2.0 will be supported or should I just alter it?

**Comment by alexmt** [comment]
Hello @TheJaySmith ,

ArgoCD v0.9.x uses ksonnet v0.11 . Master is already on ksonnet v0.13 . Release is supposed to be next week. Meanwhile, you can just change app.yaml version to '0.1.0' . 

Please let me know if you really need any ksonnet v0.13 features. I can push master to argoproj/<image>:latests to unblock up

**Comment by TheJaySmith** [comment]
Hello Alex,

Thanks for the update I think I can wait until next week for the push. No need to speed the release. I will test again in a week and report back

**Comment by jessesuen** [comment]
Closing this since this is a duplicate of the ksonnet v0.13 upgrade.
